index.js

```diff
@@ -18,6 +18,12 @@ const log = logrAll(config.log || {});
 log(['init', 'cold-start'], 'Function initialized');
 
 const logRequest = function(req) {
+  if (req.queryStringParameters) {
+    req.query = req.queryStringParameters;
```

Comment:
can we make it so it doesn't manipulate the req object?  seems like that could have downstream effects